docs/authentication/session-management-overview.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Session Management overview
+id: session-management-overview
+description: ''
+slug: /session-management-overview
+keywords:
+  - api token
+  - access token
+  - session management
+pagination_next: null
+pagination_prev: null
+last_update:
+  date: 08/08/2023
+  author: William May
+doc_type: overview
+displayed_sidebar: mainSidebar
+---
+
+import MultiLanguageCodeBlock from '@site/src/components/CodeBlocks/MultiLanguageCodeBlock';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## What is Session Management
+
+Session management is a crucial aspect of web application development. An authentication session is a period of user interaction with the application, typically starting with the user providing credentials and ending with logout or inactivity timeout. Effective session management ensures user data privacy, maintains session state, and provides a seamless user experience.
+
+The Beyond Identity session consists of something we call a `token`, specifically [access token](/docs/next/api-token-overview)s. An access token, also called an API token, is an object or file that enables access to an application or resource. The token is a JWT, which is digitally signed and contains information about what resources can be accessed and for how long.
+
```

Comment:
Do we offer short lived and long lived session tokens?

```suggestion
The Beyond Identity session consists of a `token`, specifically an [access token](/docs/next/api-token-overview), which is implemented as a JSON Web Token (JWT). An access token, also used for API authenitcation, enables access to an application or resource. The token is digitally signed and contains information about what resources can be accessed and for how long.
```

Reply:
Reading [Corbado](https://docs.corbado.com/sessions/overview)'s documentation, I believe we only support short-term sessions.